policy/apps.py

```diff
@@ -21,9 +23,18 @@
     "ACTIVATION_OPTION_CONTRIBUTION": 1,
     "CTIVATION_OPTION_PAYMENT": 2,
     "ACTIVATION_OPTION_READY": 3,
-    "contribution_receipt_length": 5
+    "contribution_receipt_length": 5,
+    "comores_features_enabled": False
```

Comment:
This should be feature specific and not implementation specific.

Reply:
Renamed to remove implementation references